update steps for cluster page changes (#11883)
pingcap/docs: tidb-cloud/backup-and-restore.md

To restore a deleted cluster from recycle bin, take the following steps:

1. Log in to the [TiDB Cloud console](https://tidbcloud.com).

> 2. In the left navigation pane, do one of the following:

Author: ```suggestion
2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
```

>     - If you have multiple projects, choose a target project, and then click **Admin** > **Recycle Bin**.

Author: ```suggestion
    - If you have multiple projects, switch to the target project, and then click **Admin** > **Recycle Bin**.
```
